internal/engine/wazevo/backend/backend_test.go

```diff
@@ -1426,9 +1426,19 @@ L1 (SSA Block: blk0):
 	clz w4?, w2?
 	rbit w27, w2?
 	clz w5?, w27
-	clz x6?, x3?
+	ins v13?.D[0], x2?
+	cnt v14?.16B, v13?.16B
+	uaddlv v15?.8B, v14?.8B
+	mov v6?.D[0], x6?
+	clz x7?, x3?
 	rbit x27, x3?
-	clz x7?, x27
+	clz x8?, x27
+	ins v10?.D[0], x3?
+	cnt v11?.16B, v10?.16B
+	uaddlv v12?.8B, v11?.8B
+	mov v9?.D[0], x9?
```

Comment:
seems incorrect (e.g. where's v12 used?)

Reply:
likely bad String() for mov 🤔  (rd printed twice)